style.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+    text-align: center;
+    width: 100%;
+    line-height: 3em;
+}
+
+#keyboard {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    border: 2px solid #6292c2;;
+    padding: 5px;
+    max-width: 900px;
+    margin: 0 auto;
+    text-align: center;
```

Comment:
unnecessary property  text-align: center;

Reply:
Okey, i agree with it

Reply:
I've just corrected it